gui/src/components/Bridges/Form.js

```diff
@@ -141,6 +142,7 @@ const formikOpts = {
   },
 
   handleSubmit(values, { props, setSubmitting }) {
+    values.url = normalizeUrl(values.url, { forceHttps: true })
```

Comment:
While HTTPS is always better on production, I don't think it's ideal for local development.

According to the github page, https://github.com/sindresorhus/normalize-url#forcehttps, this setting will change `http://localhost:1337` to https which isn't great. Can we get rid of the `forceHttps`?

Reply:
@dimroc How about the latest commit?

Reply:
```
  handleSubmit(values, { props, setSubmitting }) {
    values.url = normalizeUrl(values.url, {
      forceHttps: location.protocol === 'https:'
    })
```

Hey there @NavyAdmiral, even when working as anticipated it's still going to unexpectedly change someone's input from `http` to `https` which will surprise people. I say we remove the `forceHttps` option, and later on just have an error saying we don't allow `http` on production.

This lays out my concern:

1. no prefix -> https: ok
2. http -> https: not ok, because it mangles the users input unknowingly, and then writes it to the service and db. The user only realizes it after the fact.
3. https -> https: ok

If you want to have code to only satisfy 1, I'm ok with that.

Reply:
Yeah let's remember to add an error saying we don't allow http in future. Only normalizeUrl(values.url) should suffice then